Tools/TestWebKitAPI/Tests/WebKitCocoa/ScreenTime.mm

```diff
@@ -220,4 +221,61 @@ - (void)observeValueForKeyPath:(NSString *)keyPath ofObject:(id)object change:(N
 
     EXPECT_WK_STREQ(identifier, uuidString.get());
 }
+
+TEST(ScreenTime, EphemeralSessionOff)
+{
+    __block bool done = false;
+    __block bool suppressUsageRecording = false;
+
+    InstanceMethodSwizzler swizzler {
+        PAL::getSTWebpageControllerClass(),
+        @selector(setSuppressUsageRecording:),
+        imp_implementationWithBlock(^(id object, bool ephemeralSession) {
+            suppressUsageRecording = ephemeralSession;
```

Comment:
Nit, since the current variable name doesn't really make sense in the context of the Screen Time method:

```suggestion
        imp_implementationWithBlock(^(id object, bool value) {
            suppressUsageRecording = value;
```